addons/html_editor/static/src/main/media/media_plugin.js

```diff
@@ -208,18 +209,27 @@ export class MediaPlugin extends Plugin {
         return mediaDialogClosedPromise;
     }
 
-    async savePendingImages(editableEl = this.editable) {
-        const { resModel, resId } = this.getRecordInfo(editableEl);
+    async savePendingImages() {
         // When saving a webp, o_b64_image_to_save is turned into
         // o_modified_image_to_save by saveB64Image to request the saving
         // of the pre-converted webp resizes and all the equivalent jpgs.
-        const b64Proms = [...editableEl.querySelectorAll(".o_b64_image_to_save")].map(
+        const getClosestSavable = (el) => {
+            for (const provider of this.getResource("closest_savable_providers")) {
+                const value = provider(el);
+                if (value) {
+                    return value;
+                }
+            }
+        };
```

Comment:
It looks like a delegateTo and withSequence(20, (el) => this.editable) is the default value no ?